skimage/data/__init__.py

```diff
@@ -259,3 +259,20 @@ def rocket():
 
     """
     return load("rocket.jpg")
+
+def _install_metadata_doc():
```

Comment:
ok. Do you have a preferred format? (Just to make it right with a single bullet :) )